### ISSUE TITLE
Accessibility: Does not read the contents of the hover entirely

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26105]
Testing -186214

![image](https://github-com/microsoft/vscode/assets/3840081/91e4ea57-c30f-4362-8ba5-1e9daa08fe2f)

In the above image, it only reads: `(method) def has_id(` rest of the content is skipped.
